geoh5py/objects/geo_image.py

```diff
@@ -175,6 +174,46 @@ def default_vertices(self):
             )
         return None
 
+    @property
+    def dip(self) -> float:
+        """
+        Calculated dip of the image in degrees from the vertices position.
+        :return: the dip angle.
+        """
+        if self.vertices is None or self.rotation is None:
+            raise AttributeError("The image has no vertices")
+
+        # Get rotation matrix
+        rotation_matrix = xy_rotation_matrix(np.deg2rad(-self.rotation))
+
+        # Rotate the vertices
+        rotated_vertices = np.dot(self.vertices, rotation_matrix.T).T
```

Comment:
Usually affine transformations are written as y = Ax...

```suggestion
        rotated_vertices = rotation_matrix @ self.vertices.T
```

Reply:
Okay I did a mistake here.

The correct function should be:
np.dot(rotation_matrix, self.vertices.T).T

or :
(rotation_matrix @ self.vertices.T).T

which syntax do you prefer?
np dot was used everywhere else, that's why I used it

Reply:
The new preferred syntax for array dot operation is with @ I believe. You could also skip the last transpose if you index your array differently afterwards. But whatever, both works. I have been picky enough ;)

Reply:
Sure but vertices setter should be of shape (4, 3).
So anyway, we have to transpose it. 
I'll let the dot for this PR, and change massively the dot to @ in a next PR.